superdesk/io/commands/update_ingest.py

```diff
@@ -587,7 +587,7 @@ def ingest_item(item, provider, feeding_service, rule_set=None, routing_scheme=N
             )
             return False, []
         elif old_item and not ingest_service.should_update(old_item, item, provider):
-            logger.warning(f"Resource '{ingest_collection}' " f"item '{item[GUID_FIELD]}' is already edited in SD .")
+            logger.info(f"Resource '{ingest_collection}' " f"item '{item[GUID_FIELD]}' is already edited in SD .")
```

Comment:
actually here the reason is unknown, so you can change it to something like `item should not be updated.` or you can move the logging to events service where you know why it was rejected